apps/docs/components/reference/enrichments/cli/CliGlobalFlagsHandler.tsx

```diff
@@ -2,14 +2,32 @@ import RefSubLayout from '~/layouts/ref/RefSubLayout'
 
 import spec from '~/spec/cli_v1_commands.yaml' with { type: 'yaml' }
 import Param from '~/components/Params'
+import { isFeatureEnabled } from 'common'
+import { getCustomContent } from '~/lib/custom-content/getCustomContent'
+
+const { cliCustomProfiles } = isFeatureEnabled(['cli:custom_profiles'])
+const { cliProfile } = getCustomContent(['cli:profile'])
 
 const CliGlobalFlagsHandler = () => {
+  // Transform the flags based on feature flags
+  const processedFlags = spec.flags.map((flag: any) => {
+    if (flag.id === 'profile' && cliCustomProfiles) {
+      return {
+        id: 'profile',
+        name: `--profile ${cliProfile}`,
+        description: `use ${cliProfile} profile for connecting to Supabase API`,
+        default_value: cliProfile,
```

Comment:
```suggestion
```

I'm assuming this shouldn't have a default value, because default is no profile?